src/com/dotcms/AppContext.java

```diff
@@ -0,0 +1,13 @@
+package com.dotcms;
+
+import java.util.Enumeration;
+
+/**
+ * It is intended to be a global context
+ */
+public interface AppContext {
+
+    <T> T getAttribute(String attributeName);
```

Comment:
Not sure if the Generic is any useful at least if you do not declare it on the AppContext<T>